tests/test_connect.py

```diff
@@ -282,3 +283,12 @@
 
     with pytest.raises(SMTPConnectError):
         await client.connect()
+
+
+async def test_connect_timeout_error(hostname, port):
```

Comment:
Can you move this to `test_timeouts.py`? I've tried to collect everything timeout related there.

Reply:
Move all test into test_timeouts